### PR TITLE
Prepare 3.53.0

### PR DESCRIPTION
**Pull request by davidmreed**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+
+Changes
+
+- Orgs connected using ``cci org connect`` now keep track of which ``connected_app`` service was used to connect them, and will use the same ``connected_app`` service when getting a fresh access token, even if it isn't currently the default ``connected_app`` service. This makes it easier to use CumulusCI with multiple ``connected_app`` services. (Orgs that were connected before this update will continue to use the default ``connected_app`` service.)
+- When connecting a custom ``connected_app`` service, a default ``login_url`` can be specified (to use a login URL other than https://login.salesforce.com when connecting orgs using this ``connected_app`` service).
+- The ``deploy_marketing_cloud_package`` task now utilizes the new API endpoint for deployments.
+- CumulusCI commands are more resilient in the face of corrupted or impossible to decode org config files.
+- We added documentation about managing Robot Framework locators
+- CumulusCI will now warn Windows users if long pathname support is not set up correctly.
+- CumulusCI now ships with Snowfakery 3.0
```
Comment (davisagli) [inline HISTORY.rst:16]: Link to release notes?

File: HISTORY.rst [hunk 2]
```diff
@@ -5,15 +5,19 @@ History
 3.53.0 (2022-02-24)
 -------------------
 
+Critical Changes
+
+- CumulusCI now ships with Snowfakery 3.0. See the [Snowfakery release notes](https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v3.0.0). Snowfakery 3.0 ships with opt-in features that will become the default in June 2022; please test your recipes now with ``snowfakery_version: 3`` to ensure compatibility.
```
Comment (davisagli) [inline HISTORY.rst:10]: This is RST, not Markdown

**Comment by prescod** [comment]
Give me a minute to try and fix snowfakery.readthedocs.io so I can link to it please.

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,27 @@
 History
 =======
 
+3.53.0 (2022-02-24)
+-------------------
+
+Critical Changes
+
+- CumulusCI now ships with Snowfakery 3.0. See the `Snowfakery release notes <https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v3.0.0>`__. Snowfakery 3.0 ships with opt-in features that will become the default in June 2022; please test your recipes now with ``snowfakery_version: 3`` to ensure compatibility.
```
Comment (prescod) [inline HISTORY.rst:10]:
```suggestion
- CumulusCI now ships with Snowfakery 3.0. See the `Snowfakery release notes <https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v3.0.0>`__. Snowfakery 3.0 ships with opt-in features that will become the default in June 2022; please `test your recipes <https://snowfakery.readthedocs.io/en/latest/#snowfakery-3>`__  now with ``snowfakery_version: 3`` to ensure compatibility.
```

**Review by prescod — CHANGES_REQUESTED** [review]
Suggest to add a link about how to test recipes.

**Review by prescod — APPROVED** [review]

**Pull request merged**